includes/page-controller/class-wc-admin-page-controller.php

```diff
@@ -190,7 +190,7 @@ public function get_current_page() {
 		// If 'current_screen' hasn't fired yet, the current page calculation
 		// will fail which causes `false` to be returned for all subsquent calls.
 		if ( ! did_action( 'current_screen' ) ) {
-			_doing_it_wrong( __FUNCTION__, esc_html__( 'Current page retreival should be called on or after the `current_screen` hook.', 'woocommerce-admin' ) );
+			_doing_it_wrong( __FUNCTION__, esc_html__( 'Current page retreival should be called on or after the `current_screen` hook.', 'woocommerce-admin' ), '0.12.0' );
```

Comment:
This should use `WC_ADMIN_VERSION_NUMBER`, and the current version is `0.16.0`.

Reply:
@jeffstieler Thanks :) Based on https://developer.wordpress.org/reference/functions/_doing_it_wrong/ it should use the '0.16.0' string vs the constant which will change with each release. I have updated the PR with the correct version.

Reply:
Nevermind on the version constant - I'm still ☕️ing. 😄